.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,78 @@
+name: CI Pipeline for DoDo
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+  push:
+    branches:
+      - develop
+
+jobs:
+  test_and_build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Docker Compose
+      run: |
+        docker compose version
+
+    - name: Start Services
+      run: |
+        docker compose up -d --build
+      env:
+        POSTGRES_DB: dodo
+        POSTGRES_USER: dodo_user
+        POSTGRES_PASSWORD: dodo_password
```

Comment:
Do these env make a difference if the same are already defined in docker compose?

Reply:
You are right. These environment variables are already specified in the docker-compose.yml file, and the services should inherit them during the docker compose up process.